Bond/Bond+Arrays.swift

```diff
@@ -110,11 +110,11 @@ public class DynamicArray<T>: Dynamic<Array<T>>, SequenceType {
   }
   
   public var first: T? {
-    return value.first
+    return !isEmpty ? self[0] : nil
```

Comment:
Calling value.first will crash on the internal proxy array because we aren't allowed to get the value